specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-12-01-preview/containerregistry.json

```diff
@@ -2130,6 +2143,18 @@
         "dataEndpointEnabled": {
           "description": "Enable a single data endpoint per region for serving data.",
           "type": "boolean"
+        },
+        "publicNetworkAccess": {
+          "description": "Whether or not public network access is allowed for the container registry.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
```

Comment:
Why choose enum over a boolean property like `publicNetworkAccessEnabled` as I see there's `adminUserEnabled` and `dataEndpointEnabled` in existing properties?

Reply:
Just being consistent with all other RPs for the same feature https://github.com/Azure/azure-rest-api-specs/search?q=publicNetworkAccess